api/sys_mfa.go

```diff
@@ -0,0 +1,45 @@
+package api
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+)
+
+func (c *Sys) MFAValidate(requestID string, payload map[string]interface{}) (*Secret, error) {
```

Comment:
Isn't the only reason we provide a difference between MethodName and MethodNameWithContext because we're trying to maintain backwards compatibility for users who are already using the one that doesn't take a context?

In this case, this is a brand new method, so I thought we would just want to have MFAValidate take a context as its first arg.

Reply:
I could be convinced either way!
The motivation here was that this lives in the existing `(*client).Sys()` space where most methods have both `MethodName` and `MethodNameWithContext` whereas `(*client).Auth()` is the opposite (outside of `(*client).Auth().Token()`), so I wanted `(*client).Auth().MFALogin()` to be as close to the existing `(*client).Auth().Login()` as possible.